src/pages/ImportTool.js

```diff
@@ -18,14 +19,15 @@ import { Component } from 'src/libs/wrapped-components'
 const styles = {
   container: {
     display: 'flex', alignItems: 'flex-start', flex: 'auto',
+    backgroundImage: `url(${importBackground})`, backgroundRepeat: 'no-repeat',
+    backgroundSize: '1825px', backgroundPosition: 'left 745px top -90px',
```

Comment:
Should we be specifying `left` or `right` here?

Reply:
I switched it to left so that the background wouldn't move when resizing the window.